spec/controllers/v2/content_items_controller_spec.rb

```diff
@@ -9,6 +9,93 @@
     FactoryGirl.create(:version, target: @draft, number: 2)
   end
 
+  describe "index" do
+    let(:en_draft_content_id) { SecureRandom.uuid }
```

Comment:
This test is fine as far as it goes, but it doesn't really reflect the expected data. We will have items in multiple locales with the *same* content_id, and we need to test that particular case.